server/routes/getTokensFromLists.ts

```diff
@@ -10,7 +10,7 @@ export const getTokensFromLists = async (
   next: NextFunction
 ) => {
   try {
-    const tokens = await tokenBalanceCache.getTokens();
+    const tokens = await models.Token.findAll();
```

Comment:
We should rework the filtering below into a query `where` clause here, if possible (should be possible to do non-case-sensitive comparisons).